Index: src/test/java/IT0/UserStory1.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/java/IT0/UserStory1.java b/src/test/java/IT0/UserStory1.java
--- a/src/test/java/IT0/UserStory1.java	(revision 1437a2f089414e02187e2990a857599fcc760ef4)
+++ b/src/test/java/IT0/UserStory1.java	(date 1694796635201)
@@ -38,7 +38,7 @@
     }
 
     @Test
-    @Description("CA1")
+    @Description("Recomendación de múltiples recetas")
     public void recommendationOfMultipleRecipes() {
         final int recipesLimit = 2;
         when(scorerA.score(this.unsortedRecipes.get(0))).thenReturn(0);
@@ -52,7 +52,7 @@
     }
 
     @Test
-    @Description("CA2")
+    @Description("Recomendación  de una receta")
     public void recommendationOfOneRecipe() {
         final int expectedRecipesLimit = 1;
         when(scorerA.score(this.unsortedRecipes.get(0))).thenReturn(0);
@@ -66,7 +66,7 @@
     }
 
     @Test
-    @Description("CA3")
+    @Description("No se recomiendan recetas")
     public void recipesWithoutRecommendations() {
         List<Recipe> recommendations = chefExpress.recommend();
         when(scorerA.score(this.unsortedRecipes.get(0))).thenReturn(0);
@@ -77,7 +77,7 @@
     }
 
     @Test
-    @Description("CA4")
+    @Description("Recomendación sin recetas")
     public void recommendationWithoutRecipes() {
         chefExpress = new ChefExpress(new HashSet<>(), scorerA);
 
